cookbooks/bcpc-hadoop/recipes/graphite_queries.rb

```diff
@@ -1,6 +1,9 @@
 # Get a list of cluster nodes to monitor
 monitored_nodes_objs = get_all_nodes.select { |n| not n.hostname =~ /bootstrap/ }.compact
 
+mail_to_admin = "#{node[:bcpc][:hadoop][:zabbix][:mail_to_admin]}"
+mail_to_tenant = "#{node[:bcpc][:hadoop][:zabbix][:mail_to_tenant]}"
```

Comment:
Would not these recipient definitions be better encapsulated in the `zabbix.zbx_mail.sh.erb` script since that deals with e-mail where as here the `route_to` definitions should simply be `admin` versus `tenant` personas?